pkg/bot/lark.go

```diff
@@ -218,7 +229,12 @@ func (l *LarkBot) SayHello(e map[string]interface{}) error {
 
 	chatID, err := accessAndTypeCastToString(larkChatID, event)
 	if err != nil {
-		return err
+		return fmt.Errorf("while getting chat ID: %w", err)
```

Comment:
Not immediate, but we should think about introducing Wrap() method and use it in such cases

Reply:
Yep, now I'm using native error wrapping from Go (https://go.dev/blog/go1.13-errors) but later we can introduce some external library for syntactic sugar 👍